index.html

```diff
@@ -1981,6 +1988,52 @@ <h3 id="cryptosuiteString">The `cryptosuiteString` Datatype</h3>
           </dl>
         </section>
 
+        <section>
+          <h4 id="multibase">The `multibase` Datatype</h3>
+
+          <p>
+This specification encodes binary data into [[?MULTIBASE]]-encoded strings,
+which is useful in environments that cannot represent some binary values, such
+as ASCII-only data formats. In environments that support data types
+for string values, such as RDF [[?RDF-CONCEPTS]], [[?MULTIBASE]]-encoded
+content is indicated using a literal value whose datatype is set to
+`https://w3id.org/security#multibase`.
+          </p>
+
+          <p>
+The `multibase` datatype is defined as follows:
+          </p>
+
+          <dl>
+            <dt>The URL denoting this datatype</dt>
+            <dd>
+`https://w3id.org/security#multibase`
+            </dd>
+            <dt>The lexical space</dt>
+            <dd>
+Any string that starts with a [[?MULTIBASE]] character and the rest of
+the characters consist of allowable characters in the respective base-encoding
+alphabet.
+            </dd>
+            <dt>The value space</dt>
+            <dd>
+Any arbitrary binary data value.
```

Comment:
```suggestion
An arbitrary binary data value.
```

"Any" and "arbitrary" mean the same thing in this respect, imho...

Reply:
Agreed